.github/workflows/deploy-snapshot.yaml

```diff
@@ -32,7 +29,9 @@ jobs:
         run: ./gradlew setSnapshotVersion # Do not chain this command because it writes into a file which needs to be re-read inside the next gradle command
 
       - name: Publish Library
-        run: ./gradlew publishReleasePublicationToGithubPackagesRepository
-          -Pgpr.repository=${{ github.repository }}
-          -Pgpr.user=${{ github.actor }}
-          -Pgpr.token=${{ secrets.GITHUB_TOKEN }}
+        env:
+          ORG_GPG_PASSPHRASE: ${{ secrets.ORG_GPG_PASSPHRASE }}
+          ORG_GPG_PRIVATE_KEY: ${{ secrets.ORG_GPG_PRIVATE_KEY }}
+          ORG_OSSRH_PASSWORD: ${{ secrets.ORG_OSSRH_PASSWORD }}
+          ORG_OSSRH_USERNAME: ${{ secrets.ORG_OSSRH_USERNAME }}
```

Comment:
To be honest I have no clue, what happens with our fork once this branch is merged.
Since we don't have these secrets in the Bosch Global Organisation the workflow might start failing when this workflow is executed "locally" on our Bosch Global fork?

Reply:
Due to this, it's also quite hard to test this in any other way than simply merging it and checking if it succeeds?

Reply:
Skipping deployment now when enot in eclipse-kuksa orga as described here: https://docs.github.com/en/actions/using-jobs/using-conditions-to-control-job-execution#example-only-run-job-for-specific-repository